dali/dfu/dfuserver.cpp

```diff
@@ -222,24 +245,54 @@ int main(int argc, const char *argv[])
             if (!*q)
                 break;
         }
-        q = globals->queryProp("@MONITORQUEUE");
-        if (q&&*q) {
+        globals->getProp("@monitorQueue", monitorQueue);
+#else
+        getDfuQueueName(queue, name);
+        unsigned maxJobs = globals->getPropInt("@maxJobs", 1);
+        if (maxJobs<1)
+        {
+            OERRLOG("maxJobs is %u", maxJobs);
+            return 1;
+        }
+        VStringBuffer mask("Queue[@name=\"%s\"][1]", queue.str());
+        if (serverstatus->queryProperties()->queryPropTree(mask.str()))
```

Comment:
harmless, but per previous [comment](https://github.com/hpcc-systems/HPCC-Platform/pull/14605#discussion_r599484712), this can never be true, because each instance creates it's own status server entry.
There could and will be multiple status server nodes (from different pods), with the same Queue, if replicas>1, but the 1 belonging to this instance will never have a Queue entry until this process sets it.